ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java

```diff
@@ -145,25 +145,30 @@ public static BeaconStateWithCache get_genesis_beacon_state(
 
     } else if (epoch == next_epoch) {
 
-      long current_committees_per_epoch = get_current_epoch_committee_count(state);
-      committees_per_epoch = get_next_epoch_committee_count(state);
-      shuffling_epoch = next_epoch;
       long epochs_since_last_registry_update =
           current_epoch - state.getValidator_registry_update_epoch();
       if (registry_change) {
+        committees_per_epoch = get_next_epoch_committee_count(state);
```

Comment:
These modifications don't seem to match v0.4 of the spec.  Is there a good reason why you had to deviate?  
https://github.com/ethereum/eth2.0-specs/blob/v0.4.0/specs/core/0_beacon-chain.md#get_crosslink_committees_at_slot

Reply:
I made it to match v0.5, but can switch it back to v0.4

Reply:
Ah ok.  Ya let's keep it at v0.4 unless there is a legit functional reason to borrow from v0.5

Reply:
There is one functional change that was incorrect (line 183), I will revert the other changes